studio/schemas/schemaTypes/slug.ts

```diff
@@ -56,6 +56,8 @@ function createSlugField(source: string) {
       /*
         make slugs read-only after initial publish
         to avoid breaking shared links
+
+        if new slugs are needed, redirects can be used instead
```

Comment:
should we include this is the description (the slug description is already quite long.. perhaps we should shorten it as well)